From 908effa35abcc7000e964fda6e379fdde3d13559 Mon Sep 17 00:00:00 2001
From: Fangrui Song <maskray@google.com>
Date: Fri, 13 Sep 2019 02:18:04 +0000
Subject: [UPSTREAM][ELF] Delete a redundant assignment to
 SectionBase::assigned. NFC

LinkerScript::discard marks a section dead. It is unnecessary to set the
`assigned` bit.

llvm-svn: 371804
(cherry picked from commit 51ead00bf81cd92a868bdd1551a06ec5efdb563b)

Change-Id: I0a4b06b104ccad16dbe2e3510a6985d418f569aa
---
 lld/ELF/LinkerScript.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lld/ELF/LinkerScript.cpp b/lld/ELF/LinkerScript.cpp
index dbf705dc7f0..8d8f3b596fa 100644
--- a/lld/ELF/LinkerScript.cpp
+++ b/lld/ELF/LinkerScript.cpp
@@ -457,7 +457,6 @@ void LinkerScript::discard(ArrayRef<InputSection *> v) {
     if (s == mainPart->hashTab)
       mainPart->hashTab = nullptr;
 
-    s->assigned = false;
     s->markDead();
     discard(s->dependentSections);
   }
-- 
2.25.0.341.g760bfbb309-goog

